www-bucket/script.js

```diff
@@ -12,7 +12,19 @@ const triggers = [
   "Merge branch 'master' into branch",
   'Okta',
   'rubber domes',
-  'tri-state Booleans'
+  'tri-state Booleans',
+  'gates in edh',
+  'burgers',
+  '9 letter puzzles',
+  'proxy cards',
```

Comment:
Proxy cards are totally fine imo.

Reply:
So why'd you give me real cards

Reply:
Proxies are fine but real cards are nicer. When your deck is built around a card and the promo foil version is $1 you might as well get the bling version.